tests/e2e_suite.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.import test_suite
+
+import e2e_args
+import infra.ccf
+import time
+import json
+
+from loguru import logger as LOG
+
+
+def run(args):
+
+    hosts = ["localhost", "localhost"]
+    network = infra.ccf.Network(hosts, args.debug_nodes, args.perf_nodes)
+    network.start_and_join(args)
+
+    LOG.info(f"Running {len(test_suite.tests)} tests for {args.test_duration} seconds")
+
+    run_tests = {}
+    elapsed = args.test_duration
+
+    for test in test_suite.tests:
+        test_name = f"{test.__module__}.{test.__name__}"
```

Comment:
f"{test}" ?

Reply:
`f"{test}"` prints ` Running <function test_add_node at 0x7fbe3762d830>...`

Reply:
Ah my bad, I could have sworn it would prefix the module.